vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -226,10 +226,10 @@ void FlyoutShadowNode::createView() {
       });
 
   // Set XamlRoot on the Flyout to handle XamlIsland/AppWindow scenarios.
-  if (auto flyoutBase6 = m_flyout.try_as<winrt::IFlyoutBase6>()) {
+  if (Is19H1OrHigher()) {
```

Comment:
>Is19H1OrHigher [](start = 6, length = 14)

Question: Wasn't the whole guidance from the windows sdk team when win10 released to stop performing version checks and switch to capability checks?

Reply:
the implementation of `Is19H1OrHigher` is a capability check (checks whether a certain contract is present)

Reply:
When we're running in WinUI mode, these XAML APIs should be present even if we're running downlevel.  It seems like the helper functions could be more permissive.  Am I missing something?

---
In reply to: [444601167](https://github.com/microsoft/react-native-windows/pull/5309#discussion_r444601167) [](ancestors = 444601167)

Reply:
@kmelmon actually...yes, I think that's right. I just need to do the same thing I did in the other places of aliasing the IFooN to IFoo for winui3.

Reply:
Looks good thanks!

---
In reply to: [445265358](https://github.com/microsoft/react-native-windows/pull/5309#discussion_r445265358) [](ancestors = 445265358)